extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -524,6 +533,69 @@ export class AmpConsent extends AMP.BaseElement {
     });
   }
 
+  /**
+   * Clear cache for server side decision and then sync.
+   */
+  syncRemoteResponse_() {
+    this.getConsentRemote_().then(response => {
+      if (response) {
+        this.clearCache_(response['expireCache']);
+        // Decision from promptUI takes precedence over consent decision from response
+        if (!this.consentStateChangedViaPromptUI_) {
+          this.updateCacheIfNotNull_(
+            response['consentStateValue'],
+            response['consentString'],
+            response['consentRequired']
+          );
+        }
+      }
+    });
+  }
+
+  /**
+   * Fetch stored data, and sync with server if appropriate.
+   * @param {boolean=} expireCache
+   */
+  clearCache_(expireCache) {
```

Comment:
seems weird to pass in a boolean that you only handle when it is true.

Reply:
actually, just call it `expireCache_` as it sets a dirty bit without really clearing it.

Reply:
well, you probably can inline this method, because it will only have one line.

Reply:
Good point